docs/index.rst

```diff
@@ -42,6 +42,9 @@ Analysis types
 :doc:`markdown/pipelines/guideseq`
    Guideseq analysis
 
+:doc:`markdown/pipelines/maxquant`
```

Comment:
Where is this part, I can't find the corresponding markdown part?

Reply:
I forgot to add that file to git...I just pushed it